cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/PackCommand.java

```diff
@@ -152,9 +152,11 @@ public void execute() {
             return;
         }
 
-
+        // If project is empty
         if (isProjectEmpty(project)) {
             if (project.currentPackage().compilerPluginToml().isPresent()) {
+                // Do not throw an error if a compiler plugin is present
+            } else {
```

Comment:
We can use `isEmpty()` instead of `isPresent()`

Reply:
changed to isEmpty()